js/modules/k6/experimental/fs/cache.go

```diff
@@ -0,0 +1,86 @@
+package fs
+
+import (
+	"fmt"
+	"io"
+	"path/filepath"
+	"sync"
+
+	"github.com/spf13/afero"
```

Comment:
Considering that afero [is planned to be replaced](https://github.com/grafana/k6/issues/1079) would you mind switching to the `fsext`? (https://github.com/grafana/k6/blob/master/lib/fsext/afero_links.go)